pkg/monitoring/raid/raid.go

```diff
@@ -24,11 +24,13 @@ var log = logrus.WithField("package", "raid")
 
 type RAID struct {
 	mdstatFilePath string
+	enabled        bool
 }
 
-func CreateModule() monitoring.Module {
+func CreateModule(enabled bool) monitoring.Module {
 	return &RAID{
 		mdstatFilePath: common.HostProc("mdstat"),
+		enabled:        enabled,
```

Comment:
I think it will be more clear not to have `enabled` bool inside the module. It makes no sense to have a disabled module added to the modules list

Reply:
Disabled modules are not added to list:
```
if m.IsEnabled() {
    modules = append(modules, m)
}
```
The first var contains only an enumeration of available modules and it's constructors.

Also see my previous comment regarding this situation.

Reply:
Good, this way it seems ok